kv/kv.go

```diff
@@ -121,30 +121,50 @@ const (
 
 // Retriever is the interface wraps the basic Get and Seek methods.
 type Retriever interface {
-	// Get gets the value for key k from KV storage.
+	// Get gets the value for key k from kv store.
+	// If corresponding kv pair does not exist, it returns nil and ErrNotExist.
 	Get(k Key) ([]byte, error)
-	// Seek searches for the entry with key k in KV storage.
+	// Seek creates an Iterator positioned on the first entry that k <= entry's key.
+	// If such entry is not found, it returns a invalid Iterator with no error.
```

Comment:
It means that the Seek operation is executed with no error, but the returned Iterator is invalid.